public/app/screens/main/devices/devices.jsx

```diff
@@ -1,19 +1,39 @@
 'use strict';
 
 var React = require('react');
-var DeviceList = require('./device-list');
+var Reflux = require('reflux');
+var Room = require('./room');
 
+var roomStore = require('stores/room');
+var roomActions = require('../../../actions/room');
 
+// TODO: Rename to DevicesSection
 var DeviceRegion = React.createClass({
-	render: function () {
-		return (
-			<div>
-				<h2>My devices</h2>
-				<a className="button" href="#">+ Add device</a>
-				<DeviceList/>
-			</div>
-		);
-	}
+    mixins: [Reflux.connect(roomStore, "rooms")],
+    
+    getInitialState: function() {
+        return {rooms: roomStore.getRooms()};
+    },
+    
+    render: function () {
+        var rooms = this.state.rooms.map(function(room, i) {
+            return (
+                <Room room={room} key={i} />
+            );
+        });
+        return (
+            <div>
+                <h2>My devices</h2>
+                <a className="button" href="#">+ Add device</a>
```

Comment:
It's better to create basic markup here instead of using default foundation styles